shell/platform/common/text_range.h

```diff
@@ -2,10 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#ifndef FLUTTER_SHELL_PLATFORM_CPP_TEXT_RANGE_H_
```

Comment:
Nit: s/CPP/COMMON/

Reply:
Done. Copy/pasted from a neighbouring file but like there are a bunch more of those. Will send a separate cleanup PR in a moment.

Reply:
Sent in https://github.com/flutter/engine/pull/24424. text_input_model.h was the only one missing _COMMON_, the rest were still _COMMON_CPP_. They should all be consistent after that patch.